lib/packages/docutils/rst.nim

```diff
@@ -508,19 +508,19 @@ type
                                   # number, order of occurrence
     msgHandler: MsgHandler      # How to handle errors.
     findFile: FindFileHandler   # How to find files.
+    filename: string
+    hasToc*: bool
 
-  PSharedState = ref SharedState
+  RstSharedState* = ref SharedState
```

Comment:
don't violate the convention `type PFoo = ref Foo` though, if you do that, then also rename SharedState to RstSharedState, ditto with newSharedState => newRstSharedState (not newSharedStateRst)

if you want, you can make a separate PR with just those renames and i can approve it fast

Reply:
ok, ok, i don't argue. Done.

Reply:
> don't violate the convention type PFoo = ref Foo though

That's an old and outdated convention.